doc/jsonrpc.md

```diff
@@ -10604,11 +10604,12 @@ Capture a snapshot of the current state of a logical volume.
 
 #### Parameters
 
-Name                    | Optional | Type        | Description
------------------------ | -------- | ----------- | -----------
-lvol_name               | Required | string      | UUID or alias of the logical volume to create a snapshot from
-snapshot_name           | Required | string      | Name for the newly created snapshot
-xattrs                  | Optional | string map  | Xattrs for the newly created snapshot
+Name                      | Optional | Type        | Description
+------------------------- | -------- | ----------- | -----------
+lvol_name                 | Required | string      | UUID or alias of the logical volume to create a snapshot from
+snapshot_name             | Required | string      | Name for the newly created snapshot
+xattrs                    | Optional | string map  | Xattrs for the newly created snapshot
+enable_add_update_xattrs  | Optional | boolean     | Enable the addition and the update of xattrs to the snapshot after its creation
```

Comment:
NIT: Can we use a simpler naming here? e.g., `enable_xattrs_change`

Reply:
It would be better, you're right, but this option enable only the addition and the update of xattrs. The rename and the deletion are not possible.